api/engineapi/models.py

```diff
@@ -160,6 +160,7 @@ class RegisteredContract(Base):  # type: ignore
     __table_args__ = (
         UniqueConstraint(
             "blockchain",
+            "moonstream_user_id",
```

Comment:
Are we sure if we run migration we will not get error bacause of hitting uniqe constraint?

Reply:
Yeah, this feature was just launched into production and there are only two rows in production database under this relation currently.